Linting should be improved

```
> grocery-list@0.0.0 lint /Users/konstantinzverev/Documents/web2/grocery-list
> ./node_modules/eslint/bin/eslint.js ./


/Users/konstantinzverev/Documents/web2/grocery-list/app/tests/components/List.spec.js
  4:8  error  "ReactDOM" is defined but never used  no-unused-vars

/Users/konstantinzverev/Documents/web2/grocery-list/lib/server.js
  9:5  error  "WebpackDevServer" is defined but never used  no-unused-vars

✖ 2 problems (2 errors, 0 warnings)
```

--- a/app/tests/components/List.spec.js
+++ b/app/tests/components/List.spec.js
@@ -1,7 +1,7 @@
 /* eslint max-nested-callbacks:0 */
 
 import React from 'react';
-import ReactDOM from 'react-dom';
+// import ReactDOM from 'react-dom';
 import ReactTestUtils from 'react-addons-test-utils';
 
 import List from 'components/List';

--- a/lib/server.js
+++ b/lib/server.js
@@ -6,7 +6,7 @@
 /* eslint no-process-env:0, quote-props:0 */
 
 var webpack = require('webpack');
-var WebpackDevServer = require('webpack-dev-server');
+// var WebpackDevServer = require('webpack-dev-server');
 var reappDevTools = require('reapp-dev-tools');
 
 var webpackConfig = require('../config/webpack.config');

